presto-hive/src/test/java/com/facebook/presto/hive/parquet/ParquetTester.java

```diff
@@ -383,12 +394,13 @@ private static List toRowValue(Block rowBlock, List<Type> fieldTypes)
         return Collections.unmodifiableList(values);
     }
 
-    private static HiveClientConfig getHiveClientConfig()
+    private static HiveClientConfig getHiveClientConfig(boolean useParquetColumnNames)
```

Comment:
To be consistent with the other similar method, you can rename this as `createHiveClientConfig` and chain the method calls in the body. Please do this refactor in a separate commit.